interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/MapperTable.php

```
@@ -133,12 +141,14 @@ public function getDocuments()
-    *  @param       None
-    *  @return      $mapped_values
-    */
-    public function getMappedFields($id)
+     * Function to fetch the mapped CCDA components and forms
+     *  @param       None
+     *  @return      $mapped_values
+     */
+    /**
+     * @return non-empty-array<non-empty-array<int<0, max>, array{form_dir: mixed, form_type?: mixed, form_table?: mixed, ccda_field?: mixed, name?: mixed, class?: non-falsy-string}>>[]
+     */
+    public function getMappedFields($id): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)